src/ILCompiler/reproNative/reproNative.vcxproj

```diff
@@ -20,7 +20,8 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <!-- v140 is friendly to windbg, at the moment -->
+    <PlatformToolset>v140</PlatformToolset>
```

Comment:
Is this because of the .pdbs have back pointers to .obj files with VS2017? It would be better to pass the linker option that avoids the back pointers to .obj files. @sandreenko Do you happen to know the option?

I would love to be able to always dogfood latest VS - I do not even have pre-VS2017 on most of my machines anymore.